SampleOfferWithoutReward.sol

```diff
@@ -101,6 +136,14 @@ contract SampleOfferWithoutReward {
             paidOut += amount;
     }
 
+    function withdraw() {
+        var daysSinceSignoff = now - dateOfSignature; // TODO: Can be negative?
+        // FIXME: The amount ignores hirtorical changes to dailyWithdrawLimit.
+        var amount = (oneTimeCosts + daysSinceSignoff * dailyWithdrawLimit) - paidOut;
```

Comment:
`onetimeCosts` are not included in `paidOut` . Therefore the amount is way too high, Essentially paying out the oneTimeCosts twice.